Source/Krypton Components/Krypton.Toolkit/Translations/Converters/ToastNotificationIconStrings.cs

```diff
@@ -23,55 +51,197 @@ public ToastNotificationIconStrings()
 
         #endregion
 
-        #region Public
+        #region IsDefault
+
+        [Browsable(false)]
+        public bool IsDefault => Application.Equals(DEFAULT_APPLICATION) &&
+                                 Asterisk.Equals(DEFAULT_ASTERISK) &&
+                                 Custom.Equals(DEFAULT_CUSTOM) &&
+                                 Error.Equals(DEFAULT_ERROR) &&
+                                 Exclamation.Equals(DEFAULT_EXCLAMATION) &&
+                                 Hand.Equals(DEFAULT_HAND) &&
+                                 Information.Equals(DEFAULT_INFORMATION) &&
+                                 None.Equals(DEFAULT_NONE) &&
+                                 Ok.Equals(DEFAULT_OK) &&
+                                 Question.Equals(DEFAULT_QUESTION) &&
+                                 Shield.Equals(DEFAULT_SHIELD) &&
+                                 Stop.Equals(DEFAULT_STOP) &&
+                                 SystemApplication.Equals(DEFAULT_SYSTEM_APPLICATION) &&
+                                 SystemAsterisk.Equals(DEFAULT_SYSTEM_ASTERISK) &&
+                                 SystemError.Equals(DEFAULT_SYSTEM_ERROR) &&
+                                 SystemExclamation.Equals(DEFAULT_SYSTEM_EXCLAMATION) &&
+                                 SystemHand.Equals(DEFAULT_SYSTEM_HAND) &&
+                                 SystemInformation.Equals(DEFAULT_SYSTEM_INFORMATION) &&
+                                 SystemQuestion.Equals(DEFAULT_SYSTEM_QUESTION) &&
+                                 SystemStop.Equals(DEFAULT_SYSTEM_STOP) &&
+                                 SystemWarning.Equals(DEFAULT_SYSTEM_WARNING) &&
+                                 Warning.Equals(DEFAULT_SYSTEM_WARNING) &&
+                                 WindowsLogo.Equals(DEFAULT_WINDOWS_LOGO);
 
-        [Browsable(false)] 
-        public bool IsDefault => true;
+        #endregion
+
+        #region Public
 
+        /// <summary>Gets or sets the application icon string.</summary>
+        [Localizable(true)]
+        [Category(@"Visuals")]
```

Comment:
Everytime you have added a new designer element, I have had to add a `Reset..` and `ShouldSerialize..` methods to ensure that the default values are not always placed into the designer usage in developers applications.
So please retrofit them to all these new designer elements a well please !

Reply:
Ahh... forgot about that...

Reply:
@Smurf-IV Should these be in `ToastNotificationIconStrings` or `KryptonGlobalToolkitStrings`?

Reply:
Where-ever you have the designer element, then reflection is used on the private API's for the designer to access their states

Reply:
> Where-ever you have the designer element, then reflection is used on the private API's for the designer to access their states

The weird thing is it does have `Reset..` & `ShouldSerialize..` methods in `KryptonGlobalToolkitStrings`?

```cs
/// <summary>Gets the toast notification icon strings.</summary>
        /// <value>The toast notification icon strings.</value>
        [Category(@"Visuals")]
        [Description(@"Collection of toast notification icon strings.")]
        [MergableProperty(false)]
        [DesignerSerializationVisibility(DesignerSerializationVisibility.Content)]
        [Localizable(true)]
        public ToastNotificationIconStrings ToastNotificationIconStrings => ToastNotificationIcon;

        private bool ShouldSerializeToastNotificationIconStrings() => !ToastNotificationIcon.IsDefault;

        /// <summary>Resets the toast notification icon strings.</summary>
        public void ResetToastNotificationIconStrings() => ToastNotificationIcon.Reset();
```

Reply:
They are "Too high", reflection is done on "Every" designer element.

Reply:
Moved all these to under 'Static Strings'